### PR TITLE
feat: enableTcc, enableRemb, minParticipants.

### PR DESCRIPTION
**Pull request by gpolitis**
Depends on https://github.com/jitsi/lib-jitsi-meet/pull/730

### CODE REVIEW

**Comment by gpolitis** [comment]
@paweldomas could you PTAL?

**Comment by bgrozev** [comment]
Should we keep new options added to config.js commented out (since otherwise we can not update their value with an update)?

**Comment by gpolitis** [comment]
yes, sure, as long as it's clear that the default value is provided in Jicofo

File: config.js [hunk 1]
```diff
@@ -194,13 +194,13 @@ var config = {
 
     // Disables or enables TCC.
     // (draft-holmer-rmcat-transport-wide-cc-extensions-01)
-    enableTcc: true,
+    // enableTcc: true,
```
Comment (paweldomas) [inline config.js]: can you mention that default is in Jicofo ?

**Comment by gpolitis** [comment]
test this please

**Comment by saghul** [comment]
A (short) explanation on why someone may want to tweak these would be nice.

**Comment by gpolitis** [comment]
@saghul added, thanks. Can somebody please approve and merge this, if there are no further comments?

**Comment by saghul** [comment]
Thanks @gpolitis ! I hate to be *that guy*, but I'm afraid I don't understand why we have two options which are orthogonal, and not a switch option. I'm thinking:

`bandwidthEstimation: 'tcc', // or 'remb' or any falsey value to disable it`

If new algorithms need to be added in the future we'd add yet another config option, which *no es bueno*. Supporting another value for an already existing option is straightforward. "Oh, but what if 2 of them can be combined?" I hear you:

`bandwidthEstimation: ['tcc', 'fairy-dust', 'rainbows'],`

WDYT?

**Pull request merged**